server/authentication.js

```diff
@@ -1,15 +1,16 @@
 const expressjwt = require('express-jwt')
 const jwksRsa = require('jwks-rsa')
+const axios = require('axios')
 
 const getToken = req => {
   const tokenString = req.cookies['auth._token.auth0']
     ? req.cookies['auth._token.auth0']
     : req.headers.authorization
-  const tokenMatcher = /(?<=Bearer ).+/
-  return tokenMatcher.exec(tokenString)[0]
+  const tokenMatch = /(?<=Bearer ).+/.exec(tokenString)
+  return tokenMatch ? tokenMatch[0] : null
```

Comment:
If `tokenMatch` is an array, it could possibly return an empty array instead of a falsy value which can cause issues. Check for the array length as well just to be safe